stock_scheduler_assignation_horizon/models/procurement_group.py

```diff
@@ -38,3 +44,13 @@ def _get_moves_to_assign_domain_assignation_limited(self, domain, company):
                 [("date", "<=", max_date)],
             ]
         )
+
+    def get_global_is_moves_assignation_limited(self):
+        get_param = self.env["ir.config_parameter"].sudo().get_param
+        param = "s_scheduler_assignation_horizon.stock_horizon_move_assignation"
+        return str2bool(get_param(param, False))
+
+    def get_global_moves_limit_horizon(self):
+        get_param = self.env["ir.config_parameter"].sudo().get_param
+        param = "s_scheduler_assignation_horizon.stock_horizon_move_assignation_limit"
+        return safe_eval(get_param(param, 0))
```

Comment:
If you expect an int, use `int(get_param(param, 0))`.

Maybe:

```py
        try:
            limit = int(get_param(param, 7))
        except ValueError:
            _logger.error("Wrong value for config parameter: %s", param)
            limit = 0
        return limit
```

Reply:
Nitpicking, but IMO the logger mentioned by @ivantodorovich should use a warning level and show it will use 0. e.g
```
        try:
            param_value = get_param(param, 0)
            limit = int(param_value)
        except ValueError:
            limit = 0
            _logger.warning("Wrong value for ir.config_parameter %s: %s .Using %s as default value", % (param, param_value, limit)
        return limit
```